config/status-crd.yaml

```diff
@@ -0,0 +1,34 @@
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: clusteroperators.config.openshift.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.version
+    description: The version the operator is at.
+    name: Version
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Available")].status
+    description: Whether the operator is running and stable.
+    name: Available
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Progressing")].status
+    description: Whether the operator is processing changes.
+    name: Progressing
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Available")].lastTransitionTime
+    description: The time the operator's Available status last changed.
+    name: Since
+    type: date
+  group: config.openshift.io
+  names:
+    kind: ClusterOperator
+    listKind: ClusterOperatorList
+    plural: clusteroperators
+    singular: clusteroperator
+  scope: Cluster
+  subresources:
+    status: {}
+  version: v1
+
```

Comment:
nit: drop blank/empty lines at end of file.

Reply:
copy-paste heresy. harmless